### ISSUE TITLE
Way to store fixed position in the .json file? 

### ISSUE THREAD

**Issue by guillaumeVilar** [codeout/inet-henge#28]
Hi, 

I was wondering if there is a way to store fixed position inside the .json file? 

If I understand correctly the `positionCache` option will store the position in the browser cache. 
I would like to store a fixed position in the json file so all the users will see the same initial diagram. 

Thanks a lot for your help.

**Comment by codeout** [comment]
No, we cannot define node positions in .json file currently. But I think the same .json will generate the same initial diagram as the library places nodes according to the internal rule, not randomly, does it?

If you see different initial diagrams from the same .json, we need to think about your suggestion. 

A pull request will be always welcomed :)
